src/utils/docker/index.js

```diff
@@ -0,0 +1,25 @@
+// const tagRegEx = /^(.+\/)?(.+){1}$/;
+const imageRegEx = /\/(.*?)$/;
+const digestRegEx = /^(.+)@(.+)$/;
+const tagRegEx = /^(.+):(.+)$/;
+
+const parseImagePath = (fullImagePath) => {
+  const imageParts = imageRegEx.exec(fullImagePath);
+  return imageParts ? imageParts[1] : fullImagePath;
```

Comment:
Should we check `imageParts && imageParts.length > 0 ? imageParts[1] : fullImagePath;` ?